### ISSUE TITLE
Private keys with password cannot be added to certificates

### ISSUE THREAD

**Issue by joeribekker** [open-formulieren/open-forms#4514]
### Product versie / Product version

2.6.11

### Customer reference

Taiga LV 182

### Omschrijf het probleem / Describe the bug

Nice Catch22: DigiD config requires a certificate and allows you to enter a password. However, adding a certificate with password leads to a 500 crash because you cannot add a certificate with password.

Sentry ref: 363851

```
TypeError: Password was not given but private key is encrypted
[...]
  File "simple_certmanager/utils.py", line 21, in load_pem_x509_private_key
    return load_pem_private_key(data, password=None)
```

### Screen resolution

None

### Device

None

### OS

None

**Comment by sergei-maertens** [comment]
Tentatively putting this in implemented, but once simple-certmanager is updated I'll still have to go back to django-digid-eherkenning to incorporate the changes & then make a PR to Open Forms to bump our dependencies.

**Comment by sergei-maertens** [comment]
Published django-simple-certmanager 2.2.0 for this, work in django-digid-eherkenning is ongoing.
